src/beeware_theme/_static/beeware-theme.css

```diff
@@ -11,11 +11,23 @@ body[data-theme="light"] {
     background-color: var(--color-background-primary);
 }
 
-/* Fixes issue with background colors on auto color mode */
+/* Fixes issue with code font colors on dark mode */
```

Comment:
Say what the issue is; nobody will remember a month from now. Likewise for the similar comment below.